packages/react-image/Spec.md

```diff
@@ -0,0 +1,153 @@
+# @fluentui/react-image
+
+## Background
+
+Image is a visual representation of some content, such as a photo, illustration etc.
+
+## Prior Art
+
+- Convergence epic issue: [#16658](https://github.com/microsoft/fluentui/issues/16658)
+- Open UI research [Link](https://open-ui.org/components/image.research), PR: [#274](https://github.com/openui/open-ui/pull/274)
+
+### V8/v0 comparison
+
+Comparison done in: [#17608](https://github.com/microsoft/fluentui/pull/17608)
+
+[image@v7/v8]: https://developer.microsoft.com/en-us/fluentui#/controls/web/image
+[image@v0]: https://fluentsite.z22.web.core.windows.net/0.53.0/components/image/props
+
+| Purpose                                                                 | v7 / v8                 | v0              | Matching                      |
+| ----------------------------------------------------------------------- | ----------------------- | --------------- | ----------------------------- |
+| Additional CSS class name(s) to apply                                   | `className`             | `className`     | matching                      |
+| Specifies the cover style to be used for this image                     | `coverStyle`            | N/A             | v0 missing                    |
+| Replaces the src in case of errors                                      | `errorSrc` [DEPRECATED] | N/A             | _See `onLoadingStateChanged`_ |
+| Determines how the image is scaled and cropped to fit the frame         | `imageFit`              | `fluid`         | partially matching            |
+| Expands image frame to fill its parent container                        | `maximizeFrame`         | `fluid`         | partially matching            |
+| Optional callback for when the image load state changed                 | `onLoadingStateChanged` | N/A             | v0 missing                    |
+| Fades the image in when loaded                                          | `shouldFadeIn`          | N/A             | v0 missing                    |
+| The image starts as visible and is hidden on error                      | `shouldStartVisible`    | N/A             | v0 missing                    |
+| Provides customized styling that will layer on top of the variant rules | `styles`                | `styles`        | matching                      |
+| Theme provided by HOC                                                   | `theme`                 | `variables`     | matching                      |
+| Accessibility behavior overridden by the user                           | N/A                     | `accessibility` | v7/v8 missing                 |
+| Alternative text                                                        | N/A                     | `alt`           | v7/v8 missing                 |
+| ARIA label                                                              | N/A                     | `aria-label`    | v7/v8 missing                 |
```

Comment:
do props like `alt` and `aria-label` really not exist in v8 ? I would have thought they would just be unhandled props spread on the `img` tag

Reply:
These were added only as the v0 has "alt" and "aria-label" as component attributes on its documentation, whereas the v8 does not (I also believe they're spread in the img tag as you mentioned). Do you think I should change maybe the comment from 'n/a' to something else?

Reply:
I think removing all native html attr here would be good, and then simply stating smwhere it will be possible to use all native `img` attributes as props

Reply:
Agree, removed them. Thanks